cdap-proto/src/main/java/io/cdap/cdap/proto/id/ScheduleId.java

```diff
@@ -56,7 +56,7 @@ public ScheduleId(String namespace, String application, String schedule) {
       throw new NullPointerException("Schedule id cannot be null.");
     }
     this.application = application;
-    this.version = appId.getVersion();
+    this.version = ApplicationId.DEFAULT_VERSION;
```

Comment:
Ignore version in appId and always replace it with the default "-SNAPSHOT" version